src/app/Models/Traits/CrudTrait.php

```diff
@@ -68,8 +69,13 @@ public static function isColumnNullable($column_name)
         $conn->getDoctrineSchemaManager()->getDatabasePlatform()->registerDoctrineTypeMapping('enum', 'string');
         $conn->getDoctrineSchemaManager()->getDatabasePlatform()->registerDoctrineTypeMapping('json', 'json_array');
         $conn->getDoctrineSchemaManager()->getDatabasePlatform()->registerDoctrineTypeMapping('jsonb', 'json_array');
+        try {
```

Comment:
Avoids error on column null checking. If developer adds a crud column that does not exist in the database and want to do something with later on a mutator or something we will get error checking for column nullable if if does not exists.

I think there is no problem to allow null in the field if he does not exists in the database. If it exists we return if it's nullable or not!